src/js/components/FormField/FormField.js

```diff
@@ -225,7 +225,7 @@ const FormField = forwardRef(
             child.props.focusIndicator === undefined
           ) {
             return cloneElement(child, {
-              plain: true,
+              plain: 'noBorder',
```

Comment:
I don't see "noBorder" being used anywhere. What does setting plain to this value do?